generic/stress-ng.py

```diff
@@ -43,6 +36,28 @@ class Stressng(Test):
     :avocado: tags=cpu,memory,io,fs,privileged
     """
 
+    @staticmethod
+    def run_cmdout(cmd):
+        return process.system_output(cmd, shell=True, ignore_status=True,
+                                     sudo=True).decode("utf-8")
+
+    @staticmethod
+    def set_min_free_bytes(self):
+        """
+        Before running stress-ng stressors, if requested set the value
+        for /proc/sys/vm/min_free_kbytes to recommended value based on
+        following rule:
+
+        Recommended to be 10% of memory or 2GB, whichever is larger.
+        """
+        # calculate max of 10% of total mem or 2G
+        k_bytes = max(memory.memtotal() * 0.1, 2 * 1024 * 1024 * 1024) / 1024
```

Comment:
@sacsant is this static formula we are applying or we have the algorithm defined already for this?

Reply:
This formula is based on recommendation from development, not sure if this is officially part of Linux documentation. I will check and will add public references in the form of comment

Reply:
Discussion on correct value for min_free_kbytes still in progress. Dropping patch 2 from the series until resolution of discussion.